lm_eval/tasks/pubmedqa/pubmedqa.yaml

```
@@ -12,5 +12,8 @@ metric_list:
   - metric: acc
     aggregation: mean
     higher_is_better: true
+  - metric: brier_score
+    aggregation: mean
+    higher_is_better: true
```

```suggestion
    higher_is_better: false
```
since a perfect prediction with probability 1 would give 0.

---

good catch. Corrected now